microsoft-edge/progressive-web-apps/how-to/index.md

```diff
@@ -1,463 +1,288 @@
+For development and debugging purposes, Microsoft Edge (or a window hosting a PWA local app) permits a `localhost` web server to use the Progressive Web App (PWA) APIs without HTTPS.
 
 
 <!-- ====================================================================== -->
-## Step 3 - Create a web app manifest
-
-A _web app manifest_ is a JSON file containing metadata about your app, such as its name, description, icons, and the various operating system features it uses.
-
-See:
-* [The web app manifest](https://developer.mozilla.org/docs/Web/Progressive_web_apps/Guides/Making_PWAs_installable#the_web_app_manifest) in _Making PWAs installable_ at MDN > References > Progressive web apps > Guides.
-* [Web app manifests](https://developer.mozilla.org/docs/Web/Manifest) at MDN > References.
-
-To add an app manifest to your app:
-
-1.  In Visual Studio Code, press **Ctrl+N** to create a new file with the following content, and save the file as `manifest.json`.
-
-    ```json
-    {
-        "lang": "en-us",
-        "name": "Temperature converter app",
-        "short_name": "Temperature converter",
-        "description": "A basic temperature converter application that can convert to and from Celsius, Kelvin, and Fahrenheit",
-        "start_url": "/",
-        "background_color": "#2f3d58",
-        "theme_color": "#2f3d58",
-        "orientation": "any",
-        "display": "standalone",
-        "icons": [
-            {
-                "src": "/icon512.png",
-                "sizes": "512x512"
-            }
-        ]
-    }
-    ```
+## Front-end code files (UI code)
 
-1.  Add a 512x512 pixel app icon image named `icon512.png` to your project. You can use the [sample image](./index-images/icon512.png) for testing purposes.
+A Progressive Web App (PWA) has front-end code files that are sent by the web server to the browser on the local device.
 
-1.  In Visual Studio Code, open `index.html`, and add the following code inside the `<head>` tag.
+Front-end code is the resources needed for the app to be installed on the user's device, such as HTML, CSS, and JavaScript code.
 
-    ```html
-    <link rel="manifest" href="/manifest.json">
-    ```
+A Progressive Web App (PWA) typically includes the following front-end code files:
 
-   The above code snippet links the new web app manifest file to your website.
+* An HTML file to describe the content in your app, such as the text, images, text fields, or buttons that appear in the user interface.
 
-Your VS Code project should now look somewhat like this:
+* A CSS file to organize the HTML content in a layout, and provide styles to elements.
 
-![VS Code showing the sample PWA project, with the index.html, manifest.json, and icon files](./index-images/visual-studio-project-with-manifest.png)
+* A JavaScript file to add user interactions to your user interface.
 
+* A JSON manifest file to describe your app to the host operating system.
```

Comment:
We should also add the service worker here, even if it's described in another section below.

```suggestion
* A JSON manifest file to describe your app to the host operating system.

* A JavaScript service worker file to cache the app's front-end code files and run background tasks.
```